src/System.Management.Automation/utils/Telemetry.cs

```diff
@@ -61,6 +62,26 @@ internal enum TelemetryType
         RemoteSessionOpen,
     }
 
+    /// <summary>
+    /// Set up the telemetry initializer to mask the platform specific names.
+    /// </summary>
+    public class NameObscurerTelemetryInitializer : ITelemetryInitializer
+    {
+        // Report the platform name information as "na".
+        private const string _notavailable = "na";
```

Comment:
```suggestion
        private const string NotAvailable = "na";
```

Stylecop recommends const to use Pascal casing

Reply:
yah - i decided to be consistent with the other constants in this file and didn't think it was right to change all of them.